src/main/java/org/trips/service_framework/services/CmsService.java

```diff
@@ -253,14 +255,20 @@ private List<Sku> searchAlikeSkus(CmsSearchRequestBody body) {
 
     public Date fetchSkuExpiryDate(String skuCode, DateTime createdAt) {
```

Comment:
why the data type of expiry date is Date instead of DateTime?

Reply:
we return only the date since the expiry date is stored without a time value in the database, as the time is irrelevant here

Reply:
`java.util.Date` stores date and time both same as DateTime. Ideally LocalDate should have been used.

Reply:
got it, but while adding this method to SF, I noticed we're already using the `Date` type. 
just to minimize changes, I kept it the same as `Date`. 
changing it to DateTime would also require updates to Trove and Omnitrix. so should I keep it as is or update it?

Reply:
that fine, just wanted to check,